packages/fuel-gauge/src/auth-testing.test.ts

```diff
@@ -33,29 +33,23 @@ describe('Auth Testing', () => {
   });
 
   it('can get is_caller_external', async () => {
-    const { value } = await contractInstance.functions
-      .is_caller_external()
-      .txParams({ gasPrice, gasLimit: 10_000 })
-      .call();
+    const { value } = await contractInstance.functions.is_caller_external().call();
 
     expect(value).toBeTruthy();
   });
 
   it('can check_msg_sender [with correct id]', async () => {
     const { value } = await contractInstance.functions
       .check_msg_sender({ value: wallet.address.toB256() })
-      .txParams({ gasPrice, gasLimit: 10_000 })
+      .txParams({ gasLimit: 500 })
```

Comment:
I was able to remove `txParams` from all of the tests, barring this one. 

For this call, the `gasUsed` returned by the dry-run (452) was less than the required gas (465). I tried and tried, but I wasn't able to find out why this was the case. I've hardcoded a limit here for now.

Reply:
I've tried this locally and it seems this happening because the `dryRun` called by `getTransactionCost` is reverting due `FAILED_ASSERT_SIGNAL ( AssertFailed )`. 

Strangely enough, this error does not happen when submitting the real transaction. Therefore, since the TX does not revert, it ends up using more gas, so the gas estimation made with the `dryRun` is not accurate.

Not sure why this TX is reverting to a `dryRun` but not when it is submitted.

Most likely this is the same error that is happening [here](https://github.com/FuelLabs/fuels-ts/issues/1697)

Reply:
Could this be an issue on the client?

Reply:
@Dhaiwat10 no. This is something wrong at our side.

Reply:
@Dhaiwat10 Can we have a TODO with a link to the issue above?